File: Day 21/Aayushi-Mittal.cpp
/*
Write a program to print the pattern upto n lines
A	
B C	
D E F ... Upto n lines
*/

#include <iostream>
using namespace std;
int main()
{
  int n;
  char c='A';
  cin>>n;
  for(int i=0; i<n; i++)
  {
    for(int j=0; j<i; j++)
    {
      cout<<c<<" ";
      c++;
    }
    cout<<"\n";
  }
}

File: Day 21/Aayushi-Mittal.py
"""
Write a program to print the pattern upto n lines
A	
B C	
D E F ... Upto n lines
"""

n = int(input())
c='A';
for i in range(1, n+1):
  for j in range(1, i):
    print(c, end=" ")
    c=chr(ord(c) + 1)
  print(" ")
